core/modules/layout/css/layout.flexible.admin.css

```diff
@@ -39,14 +39,12 @@
 
 .layout-flexible-editor .flexible-row {
   padding: 0;
-  border: 1.25px dashed #0074bd;
+  border: 1.25px solid #CCC;
   margin-bottom: 15px;
   border-radius: 4px;
-  box-shadow: 3px 3px 0 0 #c6c6c6;
-}
-.layout-flexible-editor .flexible-row:hover {
-  background-color: #ffffff;
   cursor: move;
+  box-shadow: 2px 2px 0 0 #d0d0d0;
+  background-color: #f7f7f7;
```

Comment:
this could be simply `background: #f7f7f7;` (that's what it is on the manage blocks UI)

Reply:
Done

Reply:
@jenlampton - I don't know if you look, but elsewhere in the flexible layout template css, we're using background-color everywhere. So maybe I should either change it everywhere or not at all?

I already made the change, so I'll take your advice on this.

Reply:
Well, I looked again and it's only in two other places. Should I make this change to all three or leave it as it is?

Reply:
Let's not change anything else in this issue :)